tools/depends/configure.ac

```diff
@@ -264,7 +264,12 @@ case $host in
       ;;
       i*86*-linux-android*)
         if test "x$use_cpu" = "xauto"; then
-          use_cpu=$host_cpu
+          use_cpu=i686
```

Comment:
Why?

Reply:
I still do not understand why this change is necessary.

Reply:
Was there on https://developer.android.com/ndk/guides/standalone_toolchain:

___Table 2. Specifiable -march values and their resulting targets.___

Toolchain | Location
-- | --
arm | $TOOLCHAIN/**arm**-linux-androideabi/lib/
arm64 | $TOOLCHAIN/**aarch64**-linux-android/lib/
x86 | $TOOLCHAIN/**i686**-linux-android/lib/
x86_64 | $TOOLCHAIN/**x86_64**-linux-android/lib/

But you seems right, there also this:
```
Selecting Your Toolchain
Before anything else, you need to decide which processor architecture your standalone toolchain is going to target. This is done with the --arch flag.
```

Reply:
What do you think is the value of `$host_cpu`, if you configure with `--host=i686-linux-android`?

Reply:
Becomes changed seen also others are possible, see note above "But you seems right, there also this:"